spec/middleware/authentication_spec.rb

```diff
@@ -39,6 +48,16 @@
         expect(body).to eq([Napa::JsonError.new('bad_password', 'bad password').to_json])
       end
 
+      it 'returns an error message if an incorrect Password header is supplied, and a Passwords header is there' do
+        app = lambda { |env| [200, {'Content-Type' => 'application/json'}, Array.new] }
+        middleware = Napa::Middleware::Authentication.new(app)
+        env = Rack::MockRequest.env_for('/test', {'HTTP_PASSWORD' => 'incorrect', 'HTTP_PASSWORDS' => 'foo,bar'})
```

Comment:
Line is too long. [113/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.